### DISCUSSION TITLE
meshdiag explores devices that the same device is mounted under two routers

### DISCUSSION THREAD

**Discussion by aaa147147** [openthread/openthread#11037] [Q&A]
As shown in the figure, exadddr is 744dbdfffe61e9eb mounted under two routers
![image](https://github.com/user-attachments/assets/11d918d7-3089-4338-a3d0-b656311f9ba4)
So, I sent them a message and it showed that the message was successfully sent
![image](https://github.com/user-attachments/assets/768fefa5-1b84-42c5-8f4d-e54e1c3c76c6)
But I only received a message reply from one of them

**I want to know if the situation I am currently facing is normal?**

**Comment by aaa147147** [comment]
[60.log](https://github.com/user-attachments/files/18095061/60.log)
The attachment is the complete log

**Comment by aaa147147** [comment]
Can anyone help me answer this question?
meshdiag show that the same device is mounted under two routers

**Is this a normal phenomenon?**

**Comment by jwhui** [comment]
The parent child table relies on the child timeout to cleanup stale entries. If a device switches to a different parent, it is possible for the old child table entry to exist until the timeout expires.

OpenThread does implement an "inform previous parent" mechanism where a device sends an empty message to its old parent in attempt to clean up the old child entry sooner, but this is best effort.

Additional packet sniffer trace and/or OpenThread logs can help provide more visibility.

**Reply by aaa147147** [reply]
Thank you for your reply. Looking forward to updates on OpenThread. If there are any updates, please let me know if it's convenient. Thank you!
